core/src/main/java/com/linecorp/armeria/common/util/SystemInfo.java

```diff
@@ -109,6 +111,21 @@
                 JETTY_ALPN_OPTIONAL_OR_AVAILABLE = false;
             }
         }
+
+        try {
+            String osName = Ascii.toUpperCase(System.getProperty("os.name", ""));
+            if (osName.startsWith("WINDOWS")) {
+                osType = osType().WINDOWS;
+            }
+            if (osName.startsWith("LINUX")) {
```

Comment:
I think `else if` is better here
```suggestion
            else if (osName.startsWith("LINUX")) {
```